src/petals/utils/misc.py

```diff
@@ -5,3 +5,13 @@
 
 def is_dummy(tensor: torch.Tensor):
     return tensor.numel() == 0
+
+
+SPECIAL_DTYPE_SIZES = {torch.bool: 1, torch.int8: 1, torch.qint32: 4}
```

Comment:
```suggestion
SPECIAL_DTYPE_SIZES = {torch.bool: 1}
```

`int8` and `qint32` are supported by `iinfo`.

Reply:
Alas, it is not:
![image](https://github.com/bigscience-workshop/petals/assets/3491902/98efa3ca-7564-4ac6-9880-7ab7379eb9d1)